test/jats-reader.native

```diff
@@ -226,6 +226,37 @@ Pandoc
       , LineBreak
       , Str "here."
       ]
+  , Header 1 ( "statements" , [] , [] ) [ Str "Statements" ]
+  , Header
+      2
+      ( "" , [] , [] )
+      [ Str "A"
+      , Space
+      , Str "label"
+      , Space
+      , Str "for"
+      , Space
+      , Str "a"
+      , Space
+      , Str "statment."
+      , Space
+      , Str "CAP"
+      , Space
+      , Str "TITLE"
+      ]
```

Comment:
I'm ignorant of JATS, so I don't know what a document with both a label and title is supposed to look like, but this seems rather odd, just appending the two.  Is that right?  OR is that combination unlikely to come up in practice?

Reply:
There is a thorough explanation by @coryschires  [here](https://github.com/jgm/pandoc/issues/8718#issuecomment-1522320635), under "_What's the difference between label and title?_", although it is quite lengthy. 

TLDR; A label is the type of content you are displaying, including an enumerator, whilst the title is the actual name of the content you are displaying.  (See official usage from the JATS website [here](https://jats.nlm.nih.gov/archiving/tag-library/1.3/element/label.html)). A simple example is 

```
<supplement>
    <label>Table 1.</label>
    <title>Average yearly growth</title>
</supplement>
```
Or, also, from @coryschires's own post:
```
<sec>
  <label>3.</label>
  <title>Conclusions</title>
</sec>
```
So, you do indeed want title and label appended together in the Header, and it is not uncommon. 

That is the reason I suggested [here](https://github.com/jgm/pandoc/issues/8718#issuecomment-1532223808) that we could not split the parsing of "title" and "label" as separate cases, since they both need to be displayed side by side when co-ocurring.

Reply:
Thanks for the explanation. It seems to me that maybe for *pandoc*'s purposes, the label should simply be suppressed.  E.g., suppose you use pandoc to convert a JATS document to PDF via LaTeX. LaTeX will supply its own label for sections and figures, so if we include the label we'll get doubles...

Reply:
Ohhh I see. Well, if @coryschires does not have any objection, I don't mind suppressing the label in that case. @coryschires, what do you think?

Reply:
Yeah, I agree that JATS distinction between `label` and `title` is unusual and causes needless confusion. Unfortunately, I don't think there's an obviously correct answer to this question.

With that in mind, I think @jgm's plan is good. Let's suppress `<label>` and see how people respond. It seems like that would be the cleanest fit with Pandoc's AST, and we can always revisit it later.